### PR TITLE
Create subject and tasks

### PR DESCRIPTION
**Pull request by limkimhuor**

### CODE REVIEW

File: app/controllers/supervisor/subjects_controller.rb [hunk 1]
```diff
@@ -2,4 +2,21 @@ class Supervisor::SubjectsController < ApplicationController
...
+    else
+      render "new"
+    end   
+  end
+
+  private
+  def subject_params
+    params.require(:subject).permit :name, :description, :status , 
+                                    tasks_attributes: [:id, :name, :description, 
+                                                      :status, :subject_id, :_destroy]
```
Comment (dieunb) [inline app/controllers/supervisor/subjects_controller.rb]: remove `subject_id`

File: app/controllers/supervisor/tasks_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Supervisor::TasksController < ApplicationController
```
Comment (dieunb) [inline app/controllers/supervisor/tasks_controller.rb]: remove, not need

File: config/routes.rb [hunk 3]
```diff
@@ -8,5 +8,6 @@
     resources :users
     resources :courses
     resources :subjects
+    resources :tasks
```
Comment (dieunb) [inline config/routes.rb]: remove

**Comment by phornsopheak** [comment]
it's ok

**Pull request merged**